plugins/kafka/check-consumer-lag.rb

```diff
@@ -141,12 +150,12 @@ def run
         case over_or_under
         when :over
           if max_lag > threshold.to_i
-            msg = "Topics `#{max_topics}` for the group(s) `#{config[:groups]}` lag: #{max_lag} (>= #{threshold})"
+            msg = "Topics `#{max_topics}` for the group `#{group}` lag: #{max_lag} (>= #{threshold})"
```

Comment:
Here `group` variable does not exist. 
Could it be calculated a new variable max_group as it is calculated max_topics and set it here?